MonoGame.Framework/Input/GamePadCapabilities.cs

```diff
@@ -46,6 +46,10 @@ public struct GamePadCapabilities
     {
         public bool IsConnected { get; internal set; }
 
+        public string DisplayName { get; internal set; }
+
+        public string Id { get; set; }

```

Comment:
I still prefer `Identifier`.

@KonajuGames @dellis1972 @mrhelmut @Jjagg - Any better ideas for this?